pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -486,6 +486,11 @@ func checkAndPublishDhcpLeases(ctx *zedrouterContext) {
 					status.Key(), status.DisplayName,
 					ulStatus.Mac, assigned)
 				ulStatus.Assigned = assigned
+				if !assigned {
+					ulStatus.IPAddrMisMatch = true
```

Comment:
Does it make sense to put an error log alluding to the fact that that App had an IP address and now it's missing?

Otherwise LGTM

Reply:
Or I can add that comment later. We can pull this PR now to get unblocked.